File: example/src/net/SDK_V1/sample.cs
using Flagship;
var client = FlagshipBuilder.Start(
  "ENV_ID",
  "API_KEY"
);

var context = new Dictionary<string, object>();
context.Add("key", "value");
var visitor = client.NewVisitor("visitor_id", context);
await visitor.SynchronizeModifications();

var btnColorFlag = visitor.GetModification("btnColor", "red", true);

File: example/src/net/SDK_V1/sample.fs
open System.Collections.Generic
open Flagship

let client = FlagshipBuilder.Start("ENV_ID","API_KEY");

let context = new Dictionary<string, obj>();
context.Add("key", "value");

let visitor = client.NewVisitor("visitor_id", context);
let btnColorFlag = visitor.GetModification("btnColor", "red", true);
